### PR TITLE
LRCI-3783 Reinvoke on 24gb slaves only

### PR DESCRIPTION
**Pull request by michaelhashimoto**
https://liferay.atlassian.net/browse/LRCI-3783

### CODE REVIEW

**Comment by pyoo47** [comment]
@michaelhashimoto Did you and Brian discuss changing the isApply* methods back to apply* ? I remember Brian changed them in a previous pull request you sent?

File: modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/BaseBuild.java [hunk 1]
```diff
@@ -50,6 +50,63 @@
...
+		}
+
+		for (SlaveOfflineRule slaveOfflineRule :
+				SlaveOfflineRule.getSlaveOfflineRules()) {
+
+			if (!slaveOfflineRule.matches(this)) {
+				continue;
+			}
+
+			takeSlaveOffline(slaveOfflineRule);
```
Comment (michaelhashimoto) [inline modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/BaseBuild.java:101]: @pyoo47 - I forgot to ask him, but it actually does apply the slave offline rules & reinvocation rules.  As seen in this portion.

I shouldn't have returned a boolean until it was actually going to be used.  But I think it'll be confusing if it's just a *isApply*.

I can change it back if this doesn't work.   Renaming isn't too difficult.

Comment (pyoo47) [reply modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/BaseBuild.java:101]: I think Brian's definition of apply means that it has offlined the slave or reinvoked the build (depending on the type of rule), not evaluate. I think it's best if you change the naming back to what he set it to or talk it though with him before we send this in.

**Comment by michaelhashimoto** [comment]
ci:test:sf

@pyoo47 - Okay, I changed it back.  Thanks!

**Comment by pyoo47** [comment]
ci:forward

**Pull request closed without merging**